app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt

```diff
@@ -65,12 +69,51 @@ class ProfileListAdapter(
             binding.TxtImg.setText(item.name[0].toString())
             binding.TxtImg.avatarBackgroundColor = bgColors[item.colorIndex]
             binding.timeStamp.text = item.timeInstance
+            binding.pauseSwitch.isChecked = item.pauseSwitch
 
             binding.profileCard.setOnClickListener {
                 val args = Bundle()
                 args.putParcelable("Profile", item)
                 val dialog = DetailsFragment.newInstance(args)
-                dialog.show((parentView.context as FragmentActivity).supportFragmentManager, "DialogFragment")
+                dialog.show(
+                    (parentView.context as FragmentActivity).supportFragmentManager,
+                    "DialogFragment"
+                )
+            }
+            if (!item.pauseSwitch && item.ehr == Calendar.getInstance().get(Calendar.HOUR_OF_DAY) && item.emin == Calendar.getInstance().get(Calendar.MINUTE)) {
+                StoreSession.writeInt(AppConstants.BEGIN_STATUS, 0)
+                StoreSession.writeLong(AppConstants.ACTIVE_PROFILE_ID, 0)
+                
```

Comment:
@rob729 I have used these lines to cover the condition mentioned by you Now the app is working fine...Please have a look